altianalysis/run_difference_with_parallel.py

```diff
@@ -1,38 +1,36 @@
 import argparse
 import os
-import tempfile
 from pathlib import Path
 
 from joblib import Parallel, delayed
 
-from altianalysis.compute_difference import _extract_rge_alti_tile_from_stream, compute_difference_between_dtms
+from altianalysis.compute_difference import main
 
 
-def _compute_one_difference(dtm_lhd_file: str, _dir: Path, _out_dir_difference: Path):
+def _compute_one_difference(dtm_lhd_file: str, _dir: Path, _out_dir_difference: Path, second_dir=None):
 
     full_dtm_file = os.path.join(_dir, dtm_lhd_file)
     full_difference_file = os.path.join(_out_dir_difference, dtm_lhd_file)
+    _second_file = None
 
-    # compute difference for individual files
-    with tempfile.NamedTemporaryFile(suffix="_dtm_rgealti.tif", delete=False) as tmp_rge:
+    if second_dir is not None:
```

Comment:
plus robuste / générique :
```suggestion
    if second_dir:
```